pie/__init__.py

```diff
@@ -1,16 +1,21 @@
 
+
 from .utils import GitInfo
 
 try:
-    __commit__ = GitInfo(__file__).get_commit()
-except Exception:
-    import logging
-    logging.warn(
-        """
-It seems like you download `pie` instead of git-cloning it.
-We won't be able to check compatibility between pretrained models and `pie` version
-        """)
-    __commit__ = None
+    from .commit_build import COMMIT
+    __commit__ = COMMIT
+except (ImportError, SyntaxError):
```

Comment:
Hi! what's going on here? I am not sure about the SyntaxError and about the role of commit_build.py (which doesn't seem to be part of the PR). Could you explain a little? Thanks.

Reply:
I need to remove the SyntaxError. That was a developement tool I used...

Reply:
and what is the case when commit_build fails to be imported (ImportError)?

Reply:
commit_build.py is added to pie/ only at upload.
I just though I might have another way to do this without `try`, tell me what you think : 

- Check if we have git information. 
  - If yes, then we can compare sha
  - If no, check commit_build
    - commit_built.commit is not None, then we compare with this one
    - otherwise, we show "You are not using git or a pip version, we can't compare sha, etc."

We remove pie/commit_build.py from .gitignore. At upload time, we write the commit info, after upload, we remove it (aka __commit__ = None). Which will lead to the message "you are not using git nor ...".

Reply:
sounds good!

Reply:
Done !